dotCMS/src/integration-test/java/com/dotmarketing/image/focalpoint/FocalPointAPITest.java

```diff
@@ -0,0 +1,160 @@
+package com.dotmarketing.image.focalpoint;
```

Comment:
are you sure that it not missing any test here?, we have 30 files modified and just a few test

Reply:
Actually only like 10 files are modified, b/c like 5 files are related to the tests, and other ones the changes are with a flush. The new functionality that is the focal point is tested.